assets/css/sensei-learning-mode/base.scss

```diff
@@ -2,17 +2,17 @@
 $breakpoint: 783px;
 
 body {
-	--primary-color: var(--sensei-course-theme-primary-color, var(--wp--preset--color--primary, #307771));
-	--bg-color: var(--sensei-course-theme-background-color, var(--wp--preset--color--background, #FFFFFF));
+	--primary-color: var(--sensei-learning-mode-primary-color, var(--wp--preset--color--primary, #307771));
+	--bg-color: var(--sensei-learning-mode-background-color, var(--wp--preset--color--background, #FFFFFF));
 	--primary-contrast-color: var(--bg-color);
-	--text-color: var(--sensei-course-theme-foreground-color, var(--wp--preset--color--text, #1E1E1E));
+	--text-color: var(--sensei-learning-mode-foreground-color, var(--wp--preset--color--text, #1E1E1E));
 	--border-color: rgba(125, 125, 125, 0.3);
```

Comment:
I think some changes were lost in the merge, these were all changed here:
https://github.com/Automattic/sensei/pull/5758/files#diff-3ce848a738a931b14431516f745c6578e71cd8ec5229a994e7254f505fd54fbd

This is how the base should look:

```css
body {
	// Try to pick up global styles, customizer or theme colors.
	--sensei-primary-color: var(--sensei-primary-color-global, var(--sensei-course-theme-primary-color, var(--wp--preset--color--primary, #307771)));
	--sensei-background-color: var(--sensei-background-color-global, var(--sensei-course-theme-background-color, var(--wp--preset--color--background, #ffffff)));
	--sensei-primary-contrast-color: var(--sensei-background-color-global, #FFFFFF);
	--sensei-text-color: var(--sensei-text-color-global, var(--sensei-course-theme-foreground-color, var(--wp--preset--color--text, var(--wp--preset--color--foreground, #1E1E1E))));
	--border-color: rgba(125, 125, 125, 0.3);
}
```
(https://github.com/Automattic/sensei/blob/feature/learning-mode/assets/css/sensei-course-theme/base.scss)